src/schema.ts

```diff
@@ -125,21 +127,34 @@ const resolvers: any = {
       return users;
     },
     reviews: (parent: any, args: any, ctx: Context) => {
+      let sort:reviewOrderByInput = { rating: 'desc' };
+      if (args.orderByRatingAsc === true) sort = { rating: 'asc' };
+
       const filteredReviews = ctx.prisma.review.findMany({
         where: {
           OR: [{ review: { contains: args.search } }],
         },
         include: {
           movie: true,
           user: true,
-        }
+        },
+        orderBy: args.orderByRatingAsc!==undefined ? sort : null,
```

Comment:
The orderBy logic is split between this line and beginning of this function took me a while to understand what it does. I would suggest to create function to keep the logic together and make it the difference between undefined and false more clear:
```
createOrderBy= (orderByRatingAsc?: boolean) => {
  if (orderByRatingAsc === undefined) {
    return null;
  }
  return { rating: orderByRatingAsc ? 'asc' : 'desc' };
}
}
```
```suggestion
        orderBy: createOrderBy(args.orderByRatingAsc),
```